examples/webhook-signing/koa/main.ts

```diff
@@ -56,10 +57,10 @@ app.use(async (ctx, next) => {
   if (ctx.request.path === '/webhook') {
     return handleWebhook(ctx, next);
   }
-  const name = ctx.request.body?.name ?? 'world';
+  const name = ctx.request.body?.['name'] ?? 'world';
```

Comment:
This syntax is new to me, is this purposeful?

Reply:
yeah, ctx.request.body is of `unknown` type. You can't dot into `unknown` but you can index.